.github/workflows/release-cocoapods.yml

```diff
@@ -0,0 +1,38 @@
+name: Release on CocoaPods
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
```

Comment:
Do you really want a workflow_dispatch for releasing ?

Reply:
This is handy to test workflow, as publishing a release is not the easiest thing to do. Anyway, removed.